README.md

```diff
@@ -65,25 +74,23 @@ After you got your config you can authenticate the user:
 ```swift
 // AppDelegate.swift
 
-config.authenticate()
+func application(_ app: UIApplication, open url: URL, options: [UIApplicationOpenURLOptionsKey : Any] = [:]) -> Bool {
+  config?.handleOpenURL(url: url, completion: { (tokenConfig) in
+	// store access token somewhere
+	let token = token.accessToken
+
+	let _ = Octokit(tokenConfig).me() { response in
+	  	switch response {
+  		case .success(let user):
```

Comment:
I would prefer to leave the documentation like it is and just fix the compiler errors so lowercasing the enum and move to `print` instead of `println`